x-pack/platform/plugins/shared/alerting/public/pages/maintenance_windows/components/create_maintenance_windows_form.tsx

```diff
@@ -429,53 +385,52 @@ export const CreateMaintenanceWindowForm = React.memo<CreateMaintenanceWindowFor
+        <>
+          <EuiFlexItem>
+            <EuiHorizontalRule margin="xl" />
+            <UseField path="scopedQuery">
+              {() => (
+                <MaintenanceWindowScopedQuerySwitch
+                  checked={isScopedQueryEnabled}
+                  onEnabledChange={onScopeQueryToggle}
+                />
+              )}
+            </UseField>
+          </EuiFlexItem>
+          <EuiFlexItem>
+            <UseField path="scopedQuery">
+              {() => (
+                <MaintenanceWindowScopedQuery
+                  ruleTypeIds={ruleTypeIds}
+                  query={query}
+                  filters={filters}
+                  isLoading={isLoadingRuleTypes}
+                  isEnabled={isScopedQueryEnabled}
+                  errors={scopedQueryErrors}
+                  onQueryChange={onQueryChange}
+                  onFiltersChange={setFilters}
+                />
+              )}
+            </UseField>
+          </EuiFlexItem>
+        </>
+        {isScopedQueryEnabled || showMultipleSolutionsWarning ? (
           <>
             <EuiFlexItem>
               <EuiHorizontalRule margin="xl" />
-              <UseField path="scopedQuery">
-                {() => (
-                  <MaintenanceWindowScopedQuerySwitch
-                    checked={isScopedQueryEnabled}
-                    onEnabledChange={onScopeQueryToggle}
-                  />
-                )}
-              </UseField>
-            </EuiFlexItem>
-            <EuiFlexItem>
-              <UseField path="solutionId">
-                {(field) => (
-                  <MaintenanceWindowSolutionSelection
-                    isScopedQueryEnabled={isScopedQueryEnabled}
-                    isLoading={isLoadingRuleTypes}
-                    selectedSolution={solutionId}
-                    availableSolutions={availableSolutions}
-                    errors={field.errors.map((error) => error.message)}
-                    onChange={onSolutionIdChange}
-                  />
-                )}
-              </UseField>
-            </EuiFlexItem>
-            <EuiFlexItem>
-              <UseField path="scopedQuery">
-                {() => (
-                  <MaintenanceWindowScopedQuery
-                    ruleTypeIds={ruleTypeIds}
-                    query={query}
-                    filters={filters}
-                    isLoading={isLoadingRuleTypes}
-                    isEnabled={isScopedQueryEnabled}
-                    errors={scopedQueryErrors}
-                    onQueryChange={onQueryChange}
-                    onFiltersChange={setFilters}
-                  />
-                )}
-              </UseField>
+              <EuiCallOut
+                data-test-subj="maintenanceWindowMultipleSolutionsRemovedWarning"
+                title={i18n.SOLUTION_CONFIG_REMOVAL_WARNING_TITLE}
+                color="warning"
+              >
+                <p>{i18n.SOLUTION_CONFIG_REMOVAL_WARNING_SUBTITLE}</p>
```

Comment:
I know it is in the original ticket

> Show the solution warning callout regardless if the filtering is set.

But I am still unsure whether this should always be shown.

I think it would make more sense to show it only if the user is editing a MW that had a category before.

If we are moving away from the this field, the user doesn't care about this. It's just more confusion.

They already ask what the category is about and the field is still there.